app/helpers/dashboard_helper.rb

```diff
@@ -39,6 +39,7 @@ def widget_data(widget)
   end
 
   def count_reports(hosts)
+    # TODO: Replace with either just out of sync or origin specific interval
```

Comment:
this is tricky one, for ansible users the ansible interval would be more appropriate to see the load distribution within that interval

I hate to say that but what about new settings just for this widget? if that bad idea, I think out of sync would be a good compromise for now

Reply:
Do you mean a setting to choose which origin the widget should respect?

That is not only a good compromise, but a good enhancement actually.
It will result in two widgets for users using multiple configuration management tools, but that might be good thing.

Reply:
two separate widgets (or better to say widget per origin) would be good solution, unless it's very complicated

Reply:
That is now possible with the origins being registered and the reason for introducing them properly. The `default_widgets` in `Dashboard::Manager` will return widgets for all origins that allow an origin option. This part is what is WIP atm.